gto-support-db/build.gradle

```diff
@@ -14,4 +14,5 @@ dependencies {
     compile project(':gto-support-core')
 
     testCompile project(':gto-support-testing')
+    androidTestCompile 'org.hamcrest:hamcrest-library:1.3'
```

Comment:
This is only a test dependency, so it won't be used beyond just testing the build :)

With that said, there are quite a few other matchers provided in the library that may come in handy, I was actually surprised this wasn't included in the core matchers library